dialogue-clients/src/main/java/com/palantir/dialogue/clients/DialogueClients.java

```diff
@@ -66,6 +67,20 @@ public static ReloadingFactory create(Refreshable<ServicesConfigBlock> scb) {
         Channel getChannel(String serviceName);
     }
 
+    public interface NonReloadingChannelFactory {
+        Channel getNonReloadingChannel(String channelName, ClientConfiguration clientConfiguration);
+    }
+
+    public interface ClientConfigurationNonReloadingClientFactory {
```

Comment:
Wasn't this deprecated before? I think we want to keep that around so folks don't use ClientConfiguration when they have other options.

Reply:
Will readd.

Reply:
Done hopefully

Reply:
Sorry, one more nit: Should we annotate the method itself instead of the type? We should probably annotate the channel factory as well.